src/components/BlockingViews/BlockingView.tsx

```diff
@@ -101,10 +101,12 @@ function BlockingView({
     const subtitleText = useMemo(
         () => (
             <>
-                <AutoEmailLink
-                    style={[styles.textAlignCenter, subtitleStyle]}
-                    text={subtitle}
-                />
+                {subtitle && (
+                    <AutoEmailLink
+                        style={[styles.textAlignCenter, subtitleStyle]}
+                        text={subtitle}
+                    />
+                )}
```

Comment:
If we keep render the text even though subtitle is empty, an extra space can be seen.

<img width="310" alt="Screenshot 2024-07-25 at 00 16 41" src="https://github.com/user-attachments/assets/a14b34a0-1e01-4ee7-a501-20d819f1b28d">